### PR TITLE
applet.program.stusb4500_nvm: Import new applet to program STUSB4500 NVM

### PR DESCRIPTION
**Pull request by smunaut**

### CODE REVIEW

**Review by whitequark — DISMISSED** [review]
LGTM

File: software/glasgow/applet/program/stusb4500_nvm/__init__.py [hunk 1]
```diff
@@ -0,0 +1,199 @@
+# Reference: STSW-STUSB004 Documentation
```
Comment (whitequark) [inline software/glasgow/applet/program/stusb4500_nvm/__init__.py:1]: Could you upload this to the archive please?

File: software/glasgow/applet/program/stusb4500_nvm/__init__.py [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+        result = result and await self._write_regs(self.FTP_DATA_BASE, 0x00)
+        result = result and await self._write_regs(self.FTP_CTRL_0, 0x40)
+        result = result and await self._write_regs(self.FTP_CTRL_0, 0x00)
+        await asyncio.sleep(0.001)
+        result = result and await self._write_regs(self.FTP_CTRL_0, 0x40)
+        return result
+
+    async def disable(self):
+        result =            await self._write_regs(self.FTP_CTRL_0, [0x40, 0x00])
+        result = result and await self._write_regs(self.FTP_KEY, self.FTP_KEY_VALUE)
```
Comment (smunaut) [inline software/glasgow/applet/program/stusb4500_nvm/__init__.py:77]: Arf, actually just noticed this should have been 0x00 value ... (to remove the "magic value" from the register).
Doesn't impact functionality (which is why I didn't notice) but I'll push a new version with that fixed tomorrow.

File: software/glasgow/applet/program/stusb4500_nvm/__init__.py (hunk 1 above)
Comment (smunaut) [reply software/glasgow/applet/program/stusb4500_nvm/__init__.py:1]: Will do. Didn't even see the archive before.

Comment (whitequark) [reply software/glasgow/applet/program/stusb4500_nvm/__init__.py:1]: Thank you!

**Comment by smunaut** [comment]
Fixed the "disable" method and added the archive accession number

**Review by whitequark — APPROVED** [review]

**Comment by whitequark** [comment]
Thanks for the nice applet!

**Pull request merged**